meson.build

```diff
@@ -0,0 +1,57 @@
+project(
```

Comment:
You may want to commit to a specific `meson_version: '>= 0.54.0'` as you are using features from that release -- new enough that it isn't in the latest Ubuntu LTS, though it is in Ubuntu 20.10. This will let meson error out with a friendly error message, so that users know they need to acquire a newer version using e.g. pip install.

Reply:
I think my first goal is probably to just to get pmdk building with any meson version, and then come back and adjust the build to support meson 0.50.0 since I think that is what is packaged in the EPEL. Granted I am not sure how packaging for Red Hat goes if you choose to use a Meson version that isn't distributed in RHEL.

Reply:
In that case, you can trigger "all the FeatureNew warnings" by setting it to `>= 0.0.0` :D

Actual support for feature checks is probably *really* spotty before the feature checks functionality got added, but that is... old enough that only, like, Debian Jessie and Ubuntu 16.04 ship OOTB with that. And Ubuntu has a backport. :D

The earliest feature check that this PR triggers is 0.47.0, unfortunately that one is for using "feature" options in meson_options.txt which you very much want. Fortunately, it is broadly available (https://repology.org/project/meson/versions) and meets the EPEL cutoff.

Reply:
maybe I can ping hughsie on #mesonbuild for advice. HSE could have the same problem, where maybe we should target RHEL/EPEL meson version depending on packaging guidelines.